Horizon/Horizon/Sources/Features/Dashboard/DashboardAssembly.swift

```diff
@@ -26,4 +26,12 @@ final class DashboardAssembly {
     static func makeView() -> DashboardView {
         DashboardView(viewModel: .init(interactor: makeGetProgramsInteractor()))
     }
+
+#if DEBUG
+    static func makePreview() -> DashboardView {
+        let interactor = GetProgramsInteractorLive(appEnvironment: .shared)
```

Comment:
We'd need to have a preview interactor so we have mock data like this: 

`GetProgramsInteractorPreview: GetProgramsInteractor`